src/main/java/TaskList.java

```diff
@@ -45,7 +45,7 @@ public TaskList(ArrayList<Task> currentList) {
     public void addTask(Task current) {
```

Comment:
The use of this TaskList might have been overly complicated, the sole purpose is to add and delete tasks and return the arraylist formed through the storage class